runtime/testy/ansible_pb2.py

```diff
@@ -0,0 +1,166 @@
+# Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
Why are you committing compiled protobuf files??? This should be removed and added to `.gitignore`